cirq-core/cirq/value/probability.py

```diff
@@ -13,6 +13,8 @@
 # limitations under the License.
 
 """Utilities for handling probabilities."""
+import numpy as np
+from cirq.qis import to_valid_state_vector
```

Comment:
Add 
```python
if TYPE_CHECKING:
      import cirq
```

Reply:
I am getting this - `NameError: name 'TYPE_CHECKING' is not defined`